### ISSUE TITLE
Dotnet framework not core?

### ISSUE THREAD

**Issue by flashgnash** [vektor9999/NolvusDashboard#13]
This is probably more of a discussion topic but it doesn't seem to be enabled

Is the any reason dotnet framework has been used instead of core? I plan to try get this working on Linux later and the older non-cross-platform versions tend to make that harder

Genuinely curious as had thought most people had moved over to core anyway nowadays

**Comment by kristaps777** [comment]
Can this Dashboard run on Linux? I've had 0 success so far launching it.

**Comment by flashgnash** [comment]
> Can this Dashboard run on Linux? I've had 0 success so far launching it.

No, that's why I was asking why it used framework instead of core

Core might've run fine on Linux as it actually supports it

**Comment by kristaps777** [comment]
Weird, I tried this fork (https://github.com/TheAndreLuiz/NolvusDashboard) as well, but no cigar. Saw it mentioned on reddit as compatible, but it doesn't seem to be.

**Comment by TimurShaidulin** [comment]
Dashboard uses WinForms so it will never work on Linux natively. Try using proton or wine.
Porting dahsboard to linux means complete remaking from scratch because you have to use another gui library like Uno or Avalonia.

Also nolvus uses reshade and enbs, they are win only so I'm not sure why you even think about trying it on linux in the first place.

**Comment by kristaps777** [comment]
Reshade & enb run just fine on linux.

**Comment by TimurShaidulin** [comment]
> Reshade & enb run just fine on linux.

Good to hear that. But everything else is still valid.
